app/models/task.py

```diff
@@ -1,6 +1,29 @@
 from flask import current_app
+import requests, os
 from app import db
 
 
+
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime)
+    is_complete = db.Column(db.Boolean)
+
+    def to_dict(self):  
+        return {
+            "id": self.task_id, 
+            "title": self.title,
+            "description": self.description,
+            "is_complete": self.completed_at is not None
+        }
+
+    def post_message_on_slack(self):
+        url = os.environ.get("URL")
+        data = {
+            'token': os.environ.get("KEY"),
+            'channel': "task_notifications",
+            'text': f"Someone just completed the task: {self.title}"
+        }
+        requests.post(url, data)
```

Comment:
It looks like there's a small error here. The syntax for sending a request body is `requests.pos(url, data=data)` https://docs.python-requests.org/en/latest/user/quickstart/#make-a-request
```suggestion
        requests.post(url, data=data)
```